### ISSUE TITLE
Build error

### ISSUE THREAD

**Issue by eduardoboucas** [11ty/eleventy-plugin-syntaxhighlight#3]
Hi! Since installing this plugin, I'm getting this error when trying to build my site:

```
Problem writing eleventy templates:
TypeError: Cannot read property 'buildPlaceholders' of undefined
    at /Users/eduardoboucas/Sites/eduardoboucas.com/node_modules/prismjs/components/prism-php.js:118:39
    at Object.run (/Users/eduardoboucas/Sites/eduardoboucas.com/node_modules/prismjs/prism.js:456:5)
    at Object.highlight (/Users/eduardoboucas/Sites/eduardoboucas.com/node_modules/prismjs/prism.js:285:11)
    at Object.module.exports [as highlight] (/Users/eduardoboucas/Sites/eduardoboucas.com/node_modules/@11ty/eleventy-plugin-syntaxhighlight/src/markdownSyntaxHighlight.js:23:18)
    at Object.default_rules.fence (/Users/eduardoboucas/Sites/eduardoboucas.com/node_modules/markdown-it/lib/renderer.js:50:27)
    at Renderer.render (/Users/eduardoboucas/Sites/eduardoboucas.com/node_modules/markdown-it/lib/renderer.js:326:38)
    at MarkdownIt.render (/Users/eduardoboucas/Sites/eduardoboucas.com/node_modules/markdown-it/lib/index.js:543:24)
    at /Users/eduardoboucas/Sites/eduardoboucas.com/node_modules/@11ty/eleventy/src/Engines/Markdown.js:68:38
    at <anonymous>
```

It seems that the PHP highlighter is causing the issue in question. I found [this](https://github.com/PrismJS/prism/issues/1394), but not sure how useful that is in tracking down the problem.

Any thoughts are much appreciated. Happy to provide any information about my setup that you find useful.

Thanks!

**Comment by zachleat** [comment]
Confirmed, investigating.

**Comment by zachleat** [comment]
Hmm, looks like this is related to https://github.com/PrismJS/prism/issues/593

We load prismjs languages dynamically but were not doing it the correct way if Prism had internal dependencies.

Easy fix!

**Comment by zachleat** [comment]
Included in now available v1.0.5!

**Comment by eduardoboucas** [comment]
Perfect, thank you!
